### PR TITLE
Feature/add epoch field to `SendIntermediateResultRequest`

### PR DESCRIPTION
**Pull request by carpawell**
Add `epoch` field to intermediate result structure for mapping trusts in node's storages.

### CODE REVIEW

File: pkg/client/reputation.go [hunk 1]
```diff
@@ -87,11 +87,20 @@ func (c *clientImpl) SendLocalTrust(ctx context.Context, prm SendLocalTrustPrm,
 
 // SendIntermediateTrustPrm groups parameters of SendIntermediateTrust operation.
 type SendIntermediateTrustPrm struct {
-	iter uint32
+	epoch uint64
```
Comment (alexvanin) [inline pkg/client/reputation.go:90]: Can you add empty line to keep all `*Prm` structures in one style?

File: v2/reputation/types.go [hunk 2]
```diff
@@ -297,11 +297,28 @@ func (x *SendLocalTrustResponse) SetBody(v *SendLocalTrustResponseBody) {
 
 // SendIntermediateResultRequestBody is a structure of SendIntermediateResult request body.
 type SendIntermediateResultRequestBody struct {
-	iter uint32
+	epoch uint64
```
Comment (alexvanin) [inline v2/reputation/types.go:300]: One more empty line there to keep code style the same.

**Review by alexvanin — DISMISSED** [review]

File: v2/reputation/types.go (hunk 2 above)
Comment (carpawell) [reply v2/reputation/types.go:300]: added

File: pkg/client/reputation.go (hunk 1 above)
Comment (carpawell) [reply pkg/client/reputation.go:90]: added

**Review by alexvanin — APPROVED** [review]

**Pull request merged**